spec/models/manageiq/providers/hawkular/middleware_manager/middleware_server_spec.rb

```diff
@@ -21,9 +21,9 @@
   end
 
   it "#collect_live_metrics for all metrics available" do
-    start_time = Time.new(2016, 5, 17, 15, 0, 0, "+02:00")    # Fixed time for testing
-    end_time = Time.new(2016, 5, 18, 0, 0, 0, "+02:00")      # Fixed time for testing
-    interval = 3600                                         # Interval in seconds
+    start_time = Time.new(2016, 5, 17, 15, 0, 0, "+02:00")
```

Comment:
I had to record a new cassette and to define the time interval I used for the tests.
Is there a better way to read these parameters ?
I looked into examples, but I didn't find any which uses explicit time interval without having to express it in a similar way as I did on the test.